src/api/app/models/event/build.rb

```diff
@@ -3,7 +3,7 @@ class Build < Base
     self.description = 'Package has finished building'
     self.abstract_class = true
     payload_keys :project, :package, :sender, :repository, :arch, :release, :readytime, :srcmd5,
-                 :rev, :reason, :bcnt, :verifymd5, :hostarch, :starttime, :endtime, :workerid, :versrel, :previouslyfailed
+                 :rev, :reason, :bcnt, :verifymd5, :hostarch, :starttime, :endtime, :workerid, :versrel, :previouslyfailed, :cumulative_failcount
```

Comment:
**nitpick**: I don't like the name, count is always cumulative right? :smile:

Reply:
correct.
Should be resolved now :-)